lib/tools/accessibility-commands.js

```diff
@@ -0,0 +1,33 @@
+import log from '../logger.js';
+
+
+const accessibilityCommands = {};
+
+/**
+ * Describes the Accessibility tree on the device under test
+ *
+ * @returns {Array<object>} JSON formatted list of all the elements currently on screen,
+ * including their bounds and accessibility information
+ */
+accessibilityCommands.describeAll = async function describeAll () {
+  log.debug(`Describing the UI on the device '${this.udid}'`);
+  return JSON.parse(await this.exec(['ui', 'describe-all']));
```

Comment:
below error can happen.

```
$ idb ui describe-all --udid A82163AC-8AD8-4ABA-AC7E-F2A3F95593ED
Could not enable accessibility for bridge '<SimulatorBridge: 0x7fa3cd701d70>'
$ echo $?
1
```

mm, I got below in another time...
```
$ idb ui describe-all --udid 322FC4CB-5D1A-4D53-9D28-8BC0DA696310
Connection was closed
```

Reply:
same as above